README.md

```diff
@@ -161,38 +174,46 @@ console.log(newData);
 
 ## Development
 
-To run tests
+To run tests on your project in development, use the following command:
```

Comment:
```suggestion
To run tests, use the following command
```
It can be on other environments, and this doesn't run on the user's project but rather this repo itself

Reply:
updated